katharsis-core/src/main/java/io/katharsis/queryspec/repository/QuerySpecRelationshipRepository.java

```diff
@@ -0,0 +1,51 @@
+package io.katharsis.queryspec.repository;
+
+import java.io.Serializable;
+
+import io.katharsis.queryParams.QueryParams;
+import io.katharsis.queryspec.DefaultQuerySpecConverter;
+import io.katharsis.queryspec.FilterOperatorRegistry;
+import io.katharsis.queryspec.QuerySpec;
+import io.katharsis.queryspec.QuerySpecConverter;
+import io.katharsis.repository.RelationshipRepository;
+import io.katharsis.resource.registry.ResourceRegistry;
+import io.katharsis.resource.registry.ResourceRegistryAware;
+
+/**
+ * QuerySpec-based RelationshipRepository that translates incoming
+ * QueryParameters to QuerySpec for ease of use.
+ */
+public abstract class QuerySpecRelationshipRepository<T, T_ID extends Serializable, D, D_ID extends Serializable>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this generic name to match the regular expression '^[A-Z][0-9]?$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00119)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this generic name to match the regular expression '^[A-Z][0-9]?$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00119)